src/config/user_groups.cpp

```diff
@@ -432,6 +432,8 @@ void UserGroups::set(const group_info& g) {
         info["K"] = ustring_view{};
         if (g.auth_data.size() == 100)
             info["s"] = g.auth_data;
+        else
+            info["s"] = ustring_view{};
```

Comment:
This will set an empty string in the serialized config, but the value will still be there; we should just erase the key completely with:
```suggestion
            info["s"].erase();
```

Reply:
Ah no worries - will try to remember in the future 😂